ansible/roles/whiskhost/tasks/deploy.yml

```diff
@@ -2,6 +2,6 @@
 # Add user and patch kernel.
 
 - include: addUser.yml
-  when: "'prod' in deployment"
+  when: docker.user is defined
```

Comment:
Add documentation for this setting.

Also wondering: @domdom82 mentioned it to be fairly "hard" to override puncutated values in environment specific `group_vars/all` (where I believe this should be set?). I think that's why we opted for `docker_user` style variables for convenience. @domdom82  comment?